### PR TITLE
Usability changes and thumbnails

### PR DESCRIPTION
**Pull request by Jellby**
Many small changes that I found necessary and hopefully others will find useful or nice:

* Make `qdarkgraystyle` optional (I don't call it "elegant" when it doesn't follow my desktop's style).

* Show original image size in status bar (in brackets).

* Properly resize the image when toggling toolbar, etc.

* Flip pages with mouse wheel.

* Let the user choose whether next/prev page should go to next/prev file.

* Add a "page fit" option. I would actually call this "best fit", I don't see the point in the current "best fit", that's simply an unconditional fit to 80% window width.

* Show scrollbars if the page does not fit the window, and take them into account when resizing.

* Esc returns from fullscreen, and window size is restored (rather than forcing it maximized).

* Save/restore settings and window size.

* Add context (right click) menu and enable shortcuts in fullscreen.

* Dockable thumbnails view.

* Restore last page (bookmarked) also when loading from command line.

* Allow loading a directory from command line.

### CODE REVIEW

**Comment by Jellby** [comment]
I'm sorry, but I don't understand this coverage thing.

File: pynocchio/pynocchio.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def __init__(self):
         self.setStyle('Fusion')
         try:
             self.setStyleSheet(qdarkgraystyle.load_stylesheet())
-        except ImportError:
+        except NameError:
```
Comment (mstuttgart) [inline pynocchio/pynocchio.py]: @Jellby thanks for the fiz. Please, change the message to something more descriptive like: "Fix Exception class in import theme function"

Comment (Jellby) [reply pynocchio/pynocchio.py]: Is it OK to "rebase" (i.e., rewrite commits that have already been pushed)?

**Comment by Jellby** [comment]
>     * [ ]  ~Make qdarkgraystyle optional (I don't call it "elegant" when it doesn't follow my desktop's style).~
> 
> The PyQT apps no follow you desktop style. This theme ensures a uniform Pynocchio appearance in all graphic environments and the theme is compiled inside of Pynocchio. Please undo this change.
> 
> You can add a menu to user choose default theme, but 'gdarkgray' must be a default theme. ok?

It does follow the desktop style for me (maybe because I use KDE). Maybe I didn't explain the change correctly. What I did is that Pynocchio will not crash if `qdarkgraystyle` is not installed, it will continue with whatever default style the system has (which in my case is what I want). But if it is installed, it will be used, as before. Is there something wrong with this?

>     * [x]  Add a "page fit" option. I would actually call this "best fit", I don't see the point in the current "best fit", that's simply an unconditional fit to 80% window width.
>       This 'best fit' git the page to best size to read and see the images of comic.

But that depends a lot on the screen size, window size, and comic type. Maybe instead of 80% it could be something like min(100%, 1500px).

Anyway, I'm thinking of adding another option to only resize to smaller sizes (never enlarge from the original size).

> Converage is a tool to measure test coverage of you code.

Yes. What I don't understand is what the report means and how to fix it.

File: pynocchio/pynocchio.py (hunk 1 above)
Comment (mstuttgart) [reply pynocchio/pynocchio.py]: Yes, this commit: https://github.com/mstuttgart/pynocchio/pull/89/commits/083b17b9178b194aa0b66d33b0aeb8fb669233bd

**Comment by mstuttgart** [comment]
About coverage, ignore it :) 

Hum.. I understand! This change is usefull when you are work in code, but when I make a release, the theme is compiled using PyInstaller. This theme its always installed. 

You can add a option (a checkbox) to active default theme of system e disable the theme.

**Comment by Jellby** [comment]
It would also be useful if you make a release for pypi (see #90). I installed it with `python3 setup.py install --user`, which is basically the same.

**Comment by mstuttgart** [comment]
To use pypi a I will need to maintain two release is diferent places. This project has many binary files (translations files, images e etc). Using PyPi, to install and run the user will need to use terminal to run the Pynocchio executable. The release with PyInstaller compiles all library in a executable, this is very easy to maintain and install (inside a .deb file, for example).

In my experience, the PyPi is good to libs and frameworks projects. To desktop apps, is more easy compiles all in an executable.

**Pull request merged**